### PR TITLE
Moondragon

### PR DESCRIPTION
**Pull request by joshaber**
API changes in light of https://developer.github.com/changes/2014-12-08-organization-permissions-api-preview/

### CODE REVIEW

File: OctoKit/OCTClient.m [hunk 1]
```diff
@@ -241,11 +244,13 @@ - (id)initWithServer:(OCTServer *)server {
 
 	[AFHTTPRequestOperation addAcceptableStatusCodes:[NSIndexSet indexSetWithIndex:OCTClientNotModifiedStatusCode]];
 
-	NSString *stableContentType = [NSString stringWithFormat:@"application/vnd.github.%@+json", OCTClientAPIVersion];
-	NSString *previewContentType = [NSString stringWithFormat:@"application/vnd.github.%@+json", OCTClientPreviewAPIVersion];
+	NSString *baseContentType = @"application/vnd.github.%@+json";
+	NSString *stableContentType = [NSString stringWithFormat:baseContentType, OCTClientAPIVersion];
+	NSString *previewContentType = [NSString stringWithFormat:baseContentType, OCTClientPreviewAPIVersion];
```
Comment (joshaber) [inline OctoKit/OCTClient.m]: Do we still need this?

Comment (jspahrsummers) [inline OctoKit/OCTClient.m]: Doesn't look like it.

File: OctoKit/OCTClient+Organizations.m [hunk 2]
```diff
@@ -16,7 +16,7 @@
 @implementation OCTClient (Organizations)
 
 - (RACSignal *)fetchUserOrganizations {
-	return [[self enqueueUserRequestWithMethod:@"GET" relativePath:@"/orgs" parameters:nil resultClass:OCTOrganization.class] oct_parsedResults];
+	return [[self enqueueUserRequestWithMethod:@"GET" relativePath:@"/user/orgs" parameters:nil resultClass:OCTOrganization.class] oct_parsedResults];
```
Comment (jspahrsummers) [inline OctoKit/OCTClient+Organizations.m]: I'm getting 404s, because “user requests” already have this prefix.

File: OctoKit/OCTClient.m (hunk 1 above)
Comment (joshaber) [inline OctoKit/OCTClient.m]: We do, apparently still in preview.

**Comment by joshaber** [comment]
:8ball:

File: OctoKit/OCTClient.m [hunk 3]
```diff
@@ -44,8 +44,13 @@
 NSString * const OCTClientErrorMessagesKey = @"OCTClientErrorMessagesKey";
 
 NSString * const OCTClientAPIVersion = @"v3";
+
+/// See https://developer.github.com/changes/2014-12-08-removing-authorizations-token/
 NSString * const OCTClientPreviewAPIVersion = @"mirage-preview";
```
Comment (jspahrsummers) [inline OctoKit/OCTClient.m]: Can you rename this to clarify which preview API it's talking about?

**Comment by jspahrsummers** [comment]
:nine:

**Comment by joshaber** [comment]
:zero:

**Pull request merged**